docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+services:
+  database:
```

Comment:
Is a compose file required for a single service?

Reply:
Wont we need one for the volume?

Reply:
Is the volume required?

Reply:
I believe so, for the postgres database.

Reply:
Is the volume used anywhere else? If not, it's doing nothing.

Reply:
A volume is used for sharing storage between containers

Reply:
Ohkay. But there is already a Dockerfile in the root directory.

Reply:
Hmm, fine.